_data/sidebars/home_sidebar.yml

```diff
@@ -74,6 +74,14 @@ entries:
       url: /widgets/
       output: web
 
+    - title: Plugins & platform-specific code
+      url: /platform-plugins/
+      output: web
+
+    - title: Custom platform-specific code with platform channels
```

Comment:
Platform channel is the overarching concept, so we use that generally. There is actually no type called `PlatformChannel` so we don't use that (there are several specific ones, e.g. `PlatformMethodChannel`)